src/classes/Commands.ts

```diff
@@ -50,6 +50,7 @@ const ADMIN_COMMANDS: string[] = [
     '!update - Update a pricelist entry',
     '!pricecheck - Requests an item to be priced by PricesTF',
     '!expand - Uses Backpack Expanders to increase the inventory limit',
+    '!delete sku=<item sku> - Delete an item (use only sku)',
```

Comment:
I think you should only be able to delete items by assetid

Reply:
Well, this delete command is a modified "!expand" command, which by default it searches for assestids and deletes the first assetid that it found.